Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java

```diff
@@ -23,21 +21,22 @@
 import java.util.Map;
 import java.util.concurrent.atomic.AtomicBoolean;
 
-import static com.mopub.common.DataKeys.ADUNIT_FORMAT;
+import static com.mopub.network.ImpressionData.ADUNIT_FORMAT;
```

Comment:
Pls use `import static com.mopub.common.DataKeys.ADUNIT_FORMAT;`

Reply:
it's not there to use.

Reply:
yes, MoPub 5.8.0 SDK not contains `DataKeys.ADUNIT_FORMAT`.